stdlib/in_channel.mli

```diff
@@ -148,6 +148,25 @@ val input_all : t -> string
     If the same channel is read concurrently by multiple threads, the returned
     string is not guaranteed to contain contiguous characters from the input. *)
 
+val input_lines : t -> string list
+(** [input_lines ic] reads lines using {!input_line}
+    until the end of file is reached.  It returns the list of all
+    lines read, in the order they were read.  The newline characters
+    that terminate lines are not included in the returned strings.
+    Empty lines produce empty strings.
+
+    @since 5.1 *)
+
+val fold_lines : ('a -> string -> 'a) -> 'a -> t -> 'a
```

Comment:
Should be `'acc` for consistency with #11858

Reply:
Done!